<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.442211 rgba(192, 97, 123, 255), stop:0.713568 rgba(241, 134, 111, 255), stop:1 rgba(255, 187, 99, 255));

color: #3b375b;
</string>
  </property>
  <widget class="QFrame" name="Frame1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>5000</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.386935 rgba(192, 97, 123, 255), stop:0.628141 rgba(241, 134, 111, 255), stop:0.969849 rgba(255, 187, 99, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HideImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Hide Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrgnImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NewImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#212121;&quot;&gt;Hide Me :D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="SideFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>99999</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#3b375b;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>114</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.442211 rgba(192, 97, 123, 255), stop:0.713568 rgba(241, 134, 111, 255), stop:1 rgba(255, 187, 99, 255));

color: #3b375b;
</string>
  </property>
  <widget class="QFrame" name="Frame1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>5000</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.140704 rgba(126, 75, 117, 255), stop:0.386935 rgba(192, 97, 123, 255), stop:0.628141 rgba(241, 134, 111, 255), stop:0.969849 rgba(255, 187, 99, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="LoadImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveImage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HideImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #7c76aa;
color: #f3ecff;
font: bold;</string>
    </property>
    <property name="text">
     <string>Hide Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrgnImage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NewImage">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="Text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffcaf9;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#212121;&quot;&gt;Hide Me :D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="SideFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>99999</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#3b375b;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>114</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>WelcomeScreen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>79</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decode">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>128</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color:#0091a9;
color: #f3ecff;

font: bold;
padding: 6px;

min-width: 237%;
max-width: 100%;
}</string>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
